.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: Test
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - master
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@preview
+        id: cache
+        with:
+          path: ~/local/rubies
+          key: 2.6.5
+
+      - uses: clupprich/ruby-build-action@master
+        id: ruby
+        with:
+          ruby-version: 2.6.5
+          cache-available: ${{ steps.cache.outputs.cache-hit == 'true' }}
+
+      - name: Setup sqlite3
+        run: sudo apt-get install libsqlite3-dev
+
+      - name: Install dependencies
+        run: bundle install
```

Comment:
Do we need to bundle all the gems to run the specs? If not, should we flesh out the `:test` group in the Gemfile to allow us to target those specifically?

Reply:
Bundling all the gems took 4 and a half minutes on the last run.

Reply:
yes, you need them all, the stuff in dev/test that is slow is the stuff for running the specs.